ui-v2/src/components/concurrency/global-concurrency-view/data-table/data-table.test.tsx

```diff
@@ -0,0 +1,81 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { createWrapper } from "@tests/utils";
+import { describe, expect, it, vi } from "vitest";
+import { Table } from "./data-table";
+
+const MOCK_DATA = [
+	{
+		id: "0",
+		created: "2021-01-01T00:00:00Z",
+		updated: "2021-01-01T00:00:00Z",
+		active: true,
+		name: "global concurrency limit 0",
+		limit: 0,
+		active_slots: 0,
+		slot_decay_per_second: 0,
+	},
+];
+
+describe("GlobalConcurrencyLimitTable -- table", () => {
```

Comment:
Could you add a test for toggling a limit's activation?

Reply:
added